engine/runtime/src/main/java/org/enso/interpreter/node/callable/InvokeCallableNode.java

```diff
@@ -251,6 +253,33 @@ public Object invokeDynamicSymbol(
     }
   }
 
+  @Specialization(guards = "warnings.hasWarnings(warning)")
+  public Object invokeWarnings(
+      Object warning,
+      VirtualFrame callerFrame,
+      State state,
+      Object[] arguments,
+      @CachedLibrary(limit = "3") WarningsLibrary warnings) {
+    try {
+      var result = execute(
```

Comment:
It'd be better to delegate to another node instance. Add `@Cached InvokeCallableNode delegate` to argument list.

Reply:
added but get a compile error:

```
[error] C:\Repos\Enso\ide\engine\runtime\src\main\java\org\enso\interpreter\node\callable\InvokeCallableNode.java:266:1:  error: Error parsing expression 'create()': The method create(CallArgumentInfo[], DefaultsExecutionMode, ArgumentsExecutionMode) in the type InvokeCallableNodeGen is not applicable for the arguments .
[error]       @Cached InvokeCallableNode invokeCallableNode,
[error]       ^warning: Unclosed files for the types 'org.enso.interpreter.runtime.type.ConstantsGen'; these types will not undergo annotation processing
```